mesa/mesaPROTON_OC.py

```diff
@@ -100,6 +100,7 @@ def __init__(self, seed=None):
         self.oc_members_scrutinize = False
         self.facilitator_repression = False
         self.facilitator_repression_multiplier = 2.0
+        self.percentage_of_facilitators = 0.005
```

Comment:
This is called percentage but used as a probability. Either change name or use self.rng.uniform(0, 100) below.